### ISSUE TITLE
wrapper cookbook - node['rabbitmq']['deb_package_url] has not been overrided during  execution phase

### ISSUE THREAD

**Issue by chrisduong** [jjasghar/rabbitmq#270]
Hi,
In my wrapper cookbook,  I define the attribute `node['rabbitmq']['version]` is '3.5.1' and I included the recipe `rabbitmq::default` in my cookbook.
However, during execution phase of `rabbitmq::default`. the node['rabbitmq']['deb_package_url] still see as default - 3.4.4-1
```ruby
default['rabbitmq']['deb_package_url'] = "https://www.rabbitmq.com/releases/rabbitmq-server/v#{node['rabbitmq']['version']}/"
```
I don't know the purpose of this, why shouldn't we define deb_package_url is a simple variable, not a node's attribute. I believe this issue won't happen.

The only way to fix for my wrapper cookbook is to override the version in Chef Roles or to force with this code
```Chef recipe
# Hack to make sure node['rabbitmq']['version'] in recipe loaded inside dependency 'rabbitmq' cookbook
node.set['rabbitmq']['version'] = node['rabbitmq']['version']
node.save
```

**Comment by chrisduong** [comment]
Hi, 
Sorry for the noise. I haven't wrapped the cookbook correctly.
I should define needed overridden attributes in my wrapper cookbook

**Comment by yardinicwaller** [comment]
This issue was identified again in #457 and resolved in #458. If you're using rabbitmq cookbook 5.3.0 or later, changing the `rabbitmq.version` attribute will be sufficient to change the version of RabbitMQ that is installed without needing to override additional attributes.
